### PR TITLE
Install activestorage into OBS

### PR DESCRIPTION
**Pull request by hellcp-work**
Allows upload of various images to be used in the frontend. Following https://edgeguides.rubyonrails.org/active_storage_overview.html

### CODE REVIEW

**Comment by danidoni** [comment]
How comes that logo is a text? What are you going to store there?

**Comment by danidoni** [comment]
> How comes that logo is a text? What are you going to store there?

Uh, I saw the referenced PR, I understand now.

File: src/api/db/migrate/20240417082156_add_logo_to_configurations.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddLogoToConfigurations < ActiveRecord::Migration[7.0]
+  def change
+    add_column :configurations, :logo, :text
```
Comment (hennevogel) [inline src/api/db/migrate/20240417082156_add_logo_to_configurations.rb:3]: text?

Comment (hennevogel) [reply src/api/db/migrate/20240417082156_add_logo_to_configurations.rb:3]: PRs have halfs now?

**Comment by hennevogel** [comment]
This is not how you upload files in rails. Please use active storage.

**Comment by hennevogel** [comment]
> This requires a few packaging changes

The spec file is in this repo too :)

**Pull request merged**